src/xcode/ENA/ENA/Source/Services/TicketValidation/Models/ValidationDecorator/DecoratorServiceModel.swift

```diff
@@ -0,0 +1,12 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+public struct ValidationDecoratorServiceModel: Codable {
```

Comment:
Did you merged the models Artur created into your branch? Because then i think this class (and the file) gets obsolete